py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -1779,6 +1779,7 @@ def aten_ops_add(
     )
 
 
+@dynamo_tensorrt_converter(operator.mul, supports_dynamic_shapes=True)
```

Comment:
Is `operator.mul` elicited outside of dynamic Linear by any chance? As in, is there a smaller model which can result in this operator appearing

Reply:
Maybe, I couldn't reproduce a model with just this operator. The smallest one is the linear example I added. Often times, if you just trace `x*2` or `operator.mul(x, 2` it records `torch.ops.aten.mul.Tensor ` in the graph.